app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render "shared/header", title: "LOGIN", prev: root_path, value: "Log in" %>
+
+<div class="bg-[#f5f6f8] h-full pt-6">
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email, class: "sr-only" %>
+    <%= f.email_field :email, autofocus: true, placeholder: "Email", class: "w-full border border-[#E9EDF4] py-3 px-5 text-base placeholder-[#ACB6BE] outline-none focus-visible:shadow-none focus:border-primary" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password, class: "sr-only" %>
+    <%= f.password_field :password, placeholder: "Password", class: "w-full border border-[#E9EDF4] py-3 px-5 text-base placeholder-[#ACB6BE] outline-none focus-visible:shadow-none focus:border-primary" %>
+  </div>
+
+<% end %>
+
+<div class="flex flex-col items-center w-full gap-4 my-2 text-base">
+<span class="hover:underline hover:text-primary"><%= link_to "Forgot your password?", new_password_path(resource_name) %></span>
+
+<p> Not a member yet? <span class="text-primary hover:underline"> <%= link_to "Sign up", new_registration_path(resource_name) %></span>
+</p>
```

Comment:
- [ ] The deployed page has error  when I click on `sign up` or `log in` links. Please make sure that the project is deployed properly, the home and detail page works as expected.
<img width="1006" alt="Screenshot 2022-10-12 at 22 04 21" src="https://user-images.githubusercontent.com/77038610/195426794-22c4704d-ff6a-4c39-b71f-8071c9983c26.png">

Reply:
The deployment has a problem and its not working but locally it is working